test/upgradeable/behavior/ownableLock.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { Contract } from 'ethers';
+
+async function shouldBehaveOwnableLock(ismock: boolean) {
```

Comment:
This expects contracts to be available at `this.token`, so you can reuse the existing behavior.